health/checker.go

```diff
@@ -2,7 +2,7 @@ package health
 
 import "fmt"
 
-//go:generate mockery --name=Manager --filename=manager.go --output=../mocks/health
+//go:generate mockery --name=Checker --filename=checker.go --output=../mocks/health
```

Comment:
This is the main change in the PR, to unify the use of 'Checker' over 'Manager'

Reply:
ah thanks for catching this, I did the refactor but then didn't catch these